release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+k6 v0.44.0 is here 🎉! This release includes:
+
+- A new experimental webcrypto module.
+- A sampling option for the tracing module.
+- Support for z/OS Unix.
+- A memory usage improvements.
+- Bug fixes and UX improvements.
+
+Some highlights from the k6-browser module are:
+
+- `locator.click` is now asynchronous, which is a breaking change.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has been implemented so k6 can now connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported when working with the k6-browser module.
+
+## Breaking changes
+
+- [xk6-browser#790](https://github.com/grafana/xk6-browser/pull/790) Convert `locator.click` to async to have feature parity with `page.click` and `elementHandle.click`. Users must remember to work with `promise.All` and `page.waitForNavigation()` when a click action results in navigation.
+
+    A `location.click` action that doesn't result in navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await tails.click(),
+    ```
+
+    A `location.click` action that does result in a navigation can be used like so:
+    ```javascript
+    const tails = page.locator("input[value='Bet on tails!']");
+    await Promise.all([
+      page.waitForNavigation(),
+      tails.click(),
+    ]);
+    ```
+
+- [xk6-browser#817](https://github.com/grafana/xk6-browser/pull/817) We've removed `--no-sandbox` from the default Chrome launch arguments. Now Chrome will launch with a sandbox, which is a more secure way of running the browser. If you are running tests under a `root` user, the browser will no longer launch unless the `--no-sandbox` argument is supplied. You can still pass this flag when launching a new Chrome instance using the `args` parameter on `chromium.launch`:
+
+    ```javascript
+    const browser = chromium.launch({
+      args: ['no-sandbox'],
+    });
+    ```
+
+- [xk6-browser#844](https://github.com/grafana/xk6-browser/pull/844) Remove the exported `version` param from the root module. Users should from now on reference the k6 version instead of the k6-browser module version.
+- [xk6-browser#838](https://github.com/grafana/xk6-browser/pull/838) Remove first meaningful paint metric. This metric is being deprecated across all the browsers, so we've removed it.
+
+## New features
+
+### xk6-webcrypto is now a built-in module [#2884](https://github.com/grafana/k6/pull/3007)
```

Comment:
Not really certain if we should even mention `xk6-webcrypto`, especially in the heading. 

I don't think many people have noticed it and almost nothing was merged until the last few weeks so :shrug:

Reply:
Fixed in 0c34d12.